tools/clang/test/CodeGenDXIL/hlsl/objects/NodeObjects/node-object-export-1.hlsl

```diff
@@ -0,0 +1,92 @@
+// RUN: %dxc -T lib_6_x -fcgl %s | FileCheck %s  --check-prefix=FCGL
+// RUN: %dxc -T lib_6_x -Zi %s | FileCheck %s  --check-prefix=DBG
+// RUN: %dxc -T lib_6_x %s | FileCheck %s  --check-prefix=O3
+// RUN: %dxc -T lib_6_x -Od %s | FileCheck %s  --check-prefix=Od
+
+struct RECORD {
+  int X;
+};
+
+// DBG: !DISubprogram(name: "foo", linkageName: "\01?foo@@YA?AU?$DispatchNodeInputRecord@URECORD@@@@U1@@Z", scope: !1, file: !1, line: {{[0-9]+}}, type: ![[FooTy:[0-9]+]], isLocal: false, isDefinition: true, scopeLine: {{[0-9]+}}, flags: DIFlagPrototyped, isOptimized: false, function: void (%"struct.DispatchNodeInputRecord<RECORD>"*, %"struct.DispatchNodeInputRecord<RECORD>"*)* @"\01?foo@@YA?AU?$DispatchNodeInputRecord@URECORD@@@@U1@@Z")
```

Comment:
Can we add checks for `dbg.declares` for local variable `input` with the same `DIComposite` type with `DispatchNodeInputRecord<RECORD>` type?

Reply:
Added.

Reply:
I don't see the checks for dbg.declare?

Reply:
Only checked !DILocalVariable metadata. I'll add dbg.declare check.

Reply:
Fixed.